src/modules/Dashboard/features/RespondentData/RespondentDataSummary/Report/Charts/MultiScatterChart/MultiScatterChart.tsx

```diff
@@ -94,17 +99,24 @@ export const MultiScatterChart = ({
 
   return (
     <Box sx={{ height, position: 'relative' }} data-testid={dataTestid}>
-      {renderChart}
-      <ChartTooltipContainer
-        ref={tooltipRef}
-        onMouseEnter={() => {
-          isHovered.current = true;
-        }}
-        onMouseLeave={hideTooltip}
-        data-testid={dataTestid}
-      >
-        <ChartTooltip data={tooltipData} data-testid={dataTestid} />
-      </ChartTooltipContainer>
+      <StyledObserverTarget className={dataTestid} sx={observerStyles} />
```

Comment:
```suggestion
      <StyledObserverTarget className={`${dataTestid}-target`} sx={observerStyles} />
```
In my opinion it's better to not duplicate dataTestId in two places. Maybe it's good idea to make some variable and pass it to both useStaticContent targetSelector and to StyledObserverTarget.